analysis/blacklist/blacklist.go

```diff
@@ -28,8 +27,22 @@ const ritaBLBufferSize = 1000
 func BuildBlacklistedCollections(res *database.Resources) {
 	//capture the current value for the error closure below
 	currentDB := res.DB.GetSelectedDB()
-
-	blDB, err := blDB.NewMongoDB(res.System.DatabaseHost, mgosec.None, "rita-bl")
+	var err error
+	var blDatabase blDB.Handle
+	if res.System.MongoDBConfig.TLS.Enabled {
+		blDatabase, err = blDB.NewSecureMongoDB(
```

Comment:
Rita uses mgosec to dial in on its own. Then RITA-BL uses the same parsed mgosec connection details to dial in separately.